tesstrain/wrapper.py

```diff
@@ -109,7 +110,8 @@ def run(
                                current environment will be used.
     :param exposures: A list of exposure levels to use (e.g. ``[-1, 0, 1]``). If
                       unspecified, language-specific ones will be used.
-    :param point_size: Size of printed text.
+    :param point_size: Size of printed text (default 12).
```

Comment:
The defaults are irrelevant here, as you can quickly see this from the function definition already.